cfme/services/requests.py

```diff
@@ -250,19 +237,12 @@ def go_to_request(cells, partial_check=False):
         cells: Search data for the requests table.
     Returns: Success of the action.
     """
-    navigate_to(Request, 'All')
-    for page in paginator.pages():
-        try:
-            # found the row!
-            row, = fields.request_list.find_rows_by_cells(cells, partial_check)
-            sel.click(row)
-            return True
-        except ValueError:
-            # row not on this page, assume it has yet to appear
-            # it might be nice to add an option to fail at this point
-            continue
-    else:
-        # Request not found at all, can't continue
+    try:
+        row = find_request(cells, partial_check)
+        sel.click(row)
+        return True
+    except Exception as e:
```

Comment:
**Required**
Its not clear which of these calls may result in an exception that gets put in the log and ignored.

If the row isn't found, find_request returns false, which means its the sel.click throwing an exception when really the failure was in find_request.

I don't think all exceptions should be hidden here, and the point of failure should be more clear.

Consider something structured more like:
```python
row = find_request(cells, partial_check)
if row:
    sel.click(row)
    return True
else:
    return False
```